pkg/idaes_examples/common/methanol/flowsheet.py

```diff
@@ -44,9 +44,11 @@
 from idaes.models.properties.modular_properties.base.generic_reaction import \
```

Comment:
Do the flowsheets really belong here in the common files, and more generally, are they actually needed/useful at all?

Having the property packages here makes sense as they are used by multiple examples (although even there I think it would be better if they could be stored with the examples that use them), but generally the flowsheet scripts are the examples (basically a copy of the notebook in script form). Thus, I my two questions:

1. As they are basically copies of the examples, shouldn't they go in the folder with the notebooks?
2. As copies, are they really necessary? They effectively duplicate code, doubling the amount of maintenance and increasing the chance of the two versions drifting apart. Originally, I believe we added these to give code-based equivalent to the notebooks in case someone wanted to just run code, but experience suggests that is unlikely.

Reply:
Looking more closely at their usage, it seems this is a case where the flowsheet.py file is imported by the notebook rather than duplicating code, so these ones are necessary. I would suggest however that they would be better places in the examples folder.

Reply:
1. I do agree that flowsheets are less likely to be imported across multiple examples than property packages, and can move the flowsheets back to the main notebook folder if you prefer.
2. Generally, I agree with you that code should not be duplicated. However, in this case the scripts contain long methods and the notebook imports and runs them - the `main()` method essentially does what the notebook does, but the two are not the same and the notebook itself cannot run without either including or importing the methods.

Reply:
I hadn't seen your second comment - yes I agree with your suggestion and will update the imports as needed.